src/main/target/common_defaults_post.h

```diff
@@ -257,3 +257,18 @@
 #else
 #define MAX_GYRODEV_COUNT 1
 #endif
+
+#ifdef USE_VCP
+#ifndef USB_DETECT_PIN
+#define USB_DETECT_PIN NONE
+#endif
+
+#ifndef USB_MSC_BUTTON_PIN
+#define USB_MSC_BUTTON_PIN NONE
+#endif
+#if defined(USE_USB_MSC_BUTTON_IPU) && !defined(MSC_BUTTON_IPU)
```

Comment:
I think this whole conditional is broken. It selects `#else` clause if `USE_USB_MSC_BUTTON_IPU` is not defined...